cmd/gpu_plugin/rm/gpu_plugin_resource_manager.go

```diff
@@ -120,26 +136,107 @@ func NewResourceManager(skipID, fullResourceName string) (ResourceManager, error
 		skipID:           skipID,
 		fullResourceName: fullResourceName,
 		prGetClientFunc:  podresources.GetV1Client,
+		assignments:      make(map[string]PodAssignmentDetails),
+		retryTimeout:     1 * time.Second,
+		cleanupInterval:  2 * time.Minute,
 	}
 
 	klog.Info("GPU device plugin resource manager enabled")
 
+	ticker := time.NewTicker(rm.cleanupInterval)
+	quitChan := make(chan struct{})
+
+	go func() {
+		defer ticker.Stop()
```

Comment:
AFAICS this never gets called. Who writes to `quitChan`?

Reply:
Yep, it's not called. I'll remove it. goroutines seem to die on their own when k8s signals SIGTERM for the pods anyhow.

Reply:
@tkatila did you plan to drop this?

Reply:
Yes, it's dropped now.